packages/oid4vci-demo-frontend/src/router/AppRouter.tsx

```diff
@@ -1,31 +1,46 @@
-import React from 'react';
-import {Routes, Route, HashRouter} from 'react-router-dom';
-import SSICredentialIssueRequestPage from '../pages/SSICredentialIssueRequestPage';
+import React, {FunctionComponent, ReactElement, useState} from 'react'
+import {Routes, Route, HashRouter, useNavigate} from 'react-router-dom'
+import SSICredentialIssueRequestPage from '../pages/SSICredentialIssueRequestPage'
 
-import SSICredentialVerifyRequestPage from '../pages/SSICredentialVerifyRequestPage';
-import SSICredentialIssuedSuccessPage from '../pages/SSICredentialIssuedSuccessPage';
-import SSILandingPage from '../pages/SSILandingPage';
+import SSICredentialVerifyRequestPage from '../pages/SSICredentialVerifyRequestPage'
+import SSICredentialIssuedSuccessPage from '../pages/SSICredentialIssuedSuccessPage'
+import SSILandingPage from '../pages/SSILandingPage'
 
-import SSIInformationRequestPage from '../pages/SSIInformationRequestPage';
-import SSIDownloadPage from "../pages/SSIDownloadPage";
+import SSIInformationRequestPage from '../pages/SSIInformationRequestPage'
+import SSIDownloadPage from "../pages/SSIDownloadPage"
 import SSIInformationSuccessPage from "../pages/SSIInformationSuccessPage"
 import SSISelectCredentialPage from "../pages/SSISelectCredentialPage"
+import {Sequencer} from "./sequencer"
+
+export const pageMap: Record<string, any> = {
```

Comment:
Just call these routes or defaultRoutes or something. Don't suffix these types. We have a typed environment, so there is no need, unless there might be clashes